server/src/test/java/org/apache/druid/server/coordinator/LoadQueuePeonTester.java

```diff
@@ -29,7 +30,26 @@
 
   public LoadQueuePeonTester()
   {
-    super(null, null, null, null, null, null);
+    super(
+        null,
+        null,
+        null,
+        null,
+        null,
+        new TestDruidCoordinatorConfig(
```

Comment:
that class would be better with a builder, though I wouldn't ask changing it now